src/Symfony/Component/ErrorHandler/Command/ErrorDumpCommand.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\ErrorHandler\ErrorRenderer\ErrorRendererInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupInterface;
+
+/**
+ * Dump error pages to plain HTML files that can be directly served by a web server.
+ *
+ * @author Loïck Piera <pyrech@gmail.com>
+ *
+ * @final
+ */
+#[AsCommand(
+    name: 'error:dump',
+    description: 'Dump error pages to plain HTML files that can be directly served by a web server',
+)]
+class ErrorDumpCommand extends Command
+{
+    public function __construct(
+        private readonly string $defaultPath,
+        private readonly Filesystem $filesystem,
+        private readonly ErrorRendererInterface $errorRenderer,
+        private readonly ?EntrypointLookupInterface $entrypointLookup = null,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addOption('path', null, InputArgument::OPTIONAL, 'Path where to dump the error pages in')
+            ->addOption('clear', null, InputArgument::REQUIRED, 'Whether to clear the error pages before dumping new ones')
+            ->addArgument('status-codes', InputArgument::IS_ARRAY, 'Status codes to dump error pages for')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $path = $input->getOption('path') ?: $this->defaultPath;
+
+        $io = new SymfonyStyle($input, $output);
+        $io->title('Dumping error pages');
+
+        $this->dump($io, $path, $input->getArgument('status-codes'), (bool) $input->getOption('clear'));
+        $io->success(\sprintf('Error pages have been dumped in "%s".', $path));
+
+        return Command::SUCCESS;
+    }
+
+    public function dump(SymfonyStyle $io, string $path, array $statusCodes, bool $clear = false): void
+    {
+        if (!$statusCodes) {
+            $statusCodes = array_filter(array_keys(Response::$statusTexts), fn ($statusCode) => $statusCode >= 400);
+        }
+
+        if ($clear || $io->confirm(\sprintf('Do you want to clear the error pages in "%s" before dumping new ones?', $path), false)) {
```

Comment:
We could be more explicit about the fact that the whole folder will be deleted. People may have other files in it.

Reply:
Is it better now?

Reply:
Maybe
```php
       if ($clear || $io->confirm(\sprintf('Do you want to remove the directory "%s" before dumping new error pages?', $path), false)) {
```
And adapt the help message of the "clear" option?

Reply:
Changed